frontend/packages/fusion-frontend/package.json

```diff
@@ -45,7 +45,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@vaadin/common-frontend": "^22.0.0-alpha1",
+    "@vaadin/common-frontend": "^0.0.10",
```

Comment:
should this be updated also automatically in the version bump script?

Reply:
Yeah, that's why I replaced `scripts/bump/update-package-dependencies.js` with `npx lerna version` in `scripts/bump/exec.sh`

Reply:
Though for now this change is necessary because bot already changed our `package.json` files, and without this change the bootstrap will break